src/sql.rs

```diff
@@ -832,13 +843,10 @@ async fn prune_tombstones(sql: &Sql) -> Result<()> {
 ///
 /// Use this together with [`rusqlite::ParamsFromIter`] to use dynamically generated
 /// parameter lists.
-pub fn repeat_vars(count: usize) -> Result<String> {
-    if count == 0 {
-        bail!("Must have at least one repeat variable");
-    }
+pub fn repeat_vars(count: usize) -> String {
```

Comment:
reasoning for allowing `count == 0` is that the resulting `SELECT x FROM y WHERE z IN()` is valid sqlite and returns zero rows anyway, right?

i checked some calls to `repeat_vars()` additionally, seems okay anyway,  in most cases we explicitly check for non-empty before anyway.

Reply:
Yes (it's valid sqlite while in some other sql implementations it's not valid).

I put this there because otherwise the tests would fail (when Contact::get_all() is called on an unconfigured context).